python/paddle/fluid/optimizer.py

```diff
@@ -1437,6 +1446,41 @@ def apply_gradients(self, params_grads):
 
         return optimize_ops
 
+    @imperative_base.no_grad
+    def minimize(self,
```

Comment:
Better not to add new minimize method for further maintainability. If you use `self._grad_clip` to record grad_clip, can this method be removed?

Reply:
YES, It can be removed